### DISCUSSION TITLE
Can I offer multiple languages or designate locale per page?

### DISCUSSION THREAD

**Discussion by joncamfield** [mmistakes/minimal-mistakes#3603] [Q&A] [answered]
I am trying to offer a multi-lingual site. I can set locale for the entire site at _config.yml, but I don't see a way to offer either a language switcher or to specify locale on a per-page option.  Open to other solutions as well of course.

**Comment by mmistakes** [comment, marked as answer]
There's no easy way to do this unless you want to modify the theme. Even with plugins it's a bit of a pain.
See these related discussions and PR for possible solutions/advice:

- https://github.com/mmistakes/minimal-mistakes/issues/1748
- https://github.com/mmistakes/minimal-mistakes/issues/917
- https://github.com/mmistakes/minimal-mistakes/issues/612
- https://github.com/mmistakes/minimal-mistakes/pull/50
